binding/python/raftify.pyi

```diff
@@ -71,6 +72,34 @@ class ReadOnlyOption:
     in that case.
     """
 
+class Level:
+    DEBUG: Final[Any]
+    """
+    Detailed information, typically only of interest to a developer trying to diagnose a problem.
+    """
+
+    INFO: Final[Any]
+    """
+    Confirmation that things are working as expected.
+    """
+
+    WARNING: Final[Any]
+    """ 
+    An indication that something unexpected happened, or that a problem might occur in the near future (e.g. ‘disk space low’). 
+    The software is still working as expected. 
+    """
+
+    ERROR: Final[Any]
+    """ 
+    Due to a more serious problem, the software has not been able to perform some function.
+    """
+
+    CRITICAL: Final[Any]
+
+    """ 
+    A serious error, indicating that the program itself may be unable to continue running.
+    """
+
```

Comment:
Copied and pasted comments may not be suitable for us and could be unnecessary. Let's simply remove the comments.